dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataOwnerApi.kt

```diff
@@ -131,4 +132,26 @@ interface DataOwnerApi {
         @PathVariable("companyId") companyId: UUID,
         @PathVariable("userId") userId: UUID,
     )
+
+    /**
+     * A method to request the current user to become data owner for the
```

Comment:
Understand what is meant, but it sounds a bit rocky. I'll think about a suggestion for this

Reply:
Addressed. Ist the new version better?

Reply:
Yes! Perfect! Closed